test/scripts/imgtestlib.py

```diff
@@ -119,15 +119,17 @@ def dl_s3_configs(destination, distro=None, arch=None):
     print(f"⬇️ Downloading configs from {s3url}")
     # only download info.json (exclude everything, then include) files, otherwise we get manifests and whole images
     job = sp.run(["aws", "s3", "sync",
+                  "--no-progress",  # wont show progress but will print file list
                   "--exclude=*",
                   "--include=*/info.json",
                   s3url, destination],
-                 capture_output=False,
+                 capture_output=True,
                  check=False)
     ok = job.returncode == 0
     if not ok:
         print(f"⚠️ Failed to sync contents of {s3url}:")
-    return ok
+        print(job.stderr.decode())
```

Comment:
Iirc `capture_output` redirects both stdout/stderr to `job.stdout`, so this one maybe `None`

Reply:
https://docs.python.org/3/library/subprocess.html?highlight=capture_output#subprocess.run implies they're captured separately, so I decided to check to be sure:
```python
>>> import subprocess as sp
>>> c = sp.run(["ls", "doesnotexist"], capture_output=True)
>>> c.stdout
b''
>>> c.stderr
b"ls: cannot access 'doesnotexist': No such file or directory\n"
>>>
```

Reply:
Oh, sorry - TIL :) if we just use stderr here, some context may get lost, should be also include stdout?